ntrans_dataprep.py

```diff
@@ -17,36 +18,37 @@
 """
 
 
-def write_data_to_csv(
-    counted_two_grams,
-    counted_three_grams,
-    counted_four_grams,
-    counted_five_grams,
-    counted_six_grams,
-):
+def write_data_to_csv(n_to_ngrams):
     """
     Writes the N-grams to CSV files. These CSV files are read from ntrans.py
     """
-    pass
 
+    filenames = ["2-grams", "3-grams", "4-grams", "5-grams", "6-grams"]
+    pathlib.Path("ngrams").mkdir(exist_ok=True)
 
-def count_ngram_frequency(twograms, threegrams, fourgrams, fivegrams, sixgrams):
+    for n, counter in n_to_ngrams.items():
+
+        # "n - 2" to get index of "filenames"
+        file_path = "./ngrams/" + filenames[n - 2] + ".csv"
```

Comment:
```suggestion
        file_path = f"./ngrams/{n}-grams.csv"
```

And you can delete `filenames` variable.

Reply:
There's also a comment above this line of code. That can be deleted too.